pkgs/build-support/fetchhg/nix-prefetch-hg

```diff
@@ -4,79 +4,79 @@ url=$1
 rev=$2
 expHash=$3
 
-hashType=$NIX_HASH_ALGO
-if test -z "$hashType"; then
-    hashType=sha256
-fi
-if test -z "$hashFormat"; then
-    hashFormat=--base32
-fi
+hashType="${NIX_HASH_ALGO:-sha256}"
+hashFormat=${hashFormat:-"--base32"}
+rev="${rev:-tip}"
```

Comment:
Any particular reason why not change the `rev` assignment above instead? (I'm sorry the PR was sitting here for so long.)